lib/ibm/cloud/sdk/power_iaas.rb

```diff
@@ -15,7 +16,7 @@ def initialize(region, guid, token, crn, tenant)
           @region = region
           @token  = token
           @tenant = tenant
-          RestClient.log = IBM::Cloud::SDK::Logging.logger
+          RestClient.log = Logging.logger
```

Comment:
With this applied
```
       class PowerIaas < BaseService
+        include Logging
+
         # Create an API Client object for the Power IaaS service
         #
         # @param region [String] the IBM Power Cloud instance region
@@ -16,7 +19,7 @@ module IBM
           @region = region
           @token  = token
           @tenant = tenant
-          RestClient.log = Logging.logger
+          RestClient.log = logger
```

I'm getting
```
>> IBM::Cloud::SDK::PowerIaas.new("a", "b", "c", "d", "e")
Traceback (most recent call last):
        3: from /home/grare/adam/src/manageiq/ibm-cloud-sdk-ruby/lib/ibm/cloud/sdk/power_iaas.rb:22:in `initialize'
        2: from /home/grare/adam/src/manageiq/ibm-cloud-sdk-ruby/lib/ibm/cloud/sdk/logging.rb:14:in `logger'
        1: from /home/grare/adam/src/manageiq/ibm-cloud-sdk-ruby/lib/ibm/cloud/sdk/logging.rb:9:in `logger'
NameError (uninitialized constant IBM::Cloud::SDK::NullLogger)
```

Reply:
Seems like we need specs for PowerIaas class.